### PR TITLE
Update utils2.ts

### PR DESCRIPTION
**Pull request by dehao0916**

### CODE REVIEW

File: src/utils2.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export function log1() {
-  console.log('=== log1 ===');
+  console.log('=== log2 ===');
```
Comment (dehao0916) [inline src/utils2.ts:2]: test1

Comment (dehao0916) [reply src/utils2.ts:2]: test2

**Comment by Dylan0916** [comment]
test1

File: src/utils2.ts (hunk 1 above)
Comment (Dylan0916) [reply src/utils2.ts:2]: Dylan 測試 thread

**Comment by Dylan0916** [comment]
Dylan 測試 issue comment

File: src/utils2.ts (hunk 1 above)
Comment (dylan-funnow) [reply src/utils2.ts:2]: Dylan-funnow 測試 thread

Comment (dehao0916) [reply src/utils2.ts:2]: PR owner 測試 thread

Comment (dehao0916) [reply src/utils2.ts:2]: PR owner 測試 thread 2

Comment (Dylan0916) [reply src/utils2.ts:2]: Dylan 測試 thread 2

**Comment by dehao0916** [comment]
PR owner 測試 issue comment

**Comment by dylan-funnow** [comment]
Dylan-funnow 測試 issue comment

**Comment by dehao0916** [comment]
PR owner 測試 issue comment 2

**Comment by dehao0916** [comment]
PR owner 測試 issue comment 3

File: src/utils2.ts (hunk 1 above)
Comment (dehao0916) [reply src/utils2.ts:2]: PR owner 測試 thread 3

Comment (Dylan0916) [reply src/utils2.ts:2]: Dylan 測試 thread 3

Comment (Dylan0916) [reply src/utils2.ts:2]: Dylan 測試 thread 4

Comment (Dylan0916) [reply src/utils2.ts:2]: Dylan 測試 thread 5

Comment (Dylan0916) [reply src/utils2.ts:2]: Dylan 測試 thread 6

Comment (Dylan0916) [reply src/utils2.ts:2]: Dylan 測試 thread 6

Comment (Dylan0916) [reply src/utils2.ts:2]: 6

Comment (Dylan0916) [reply src/utils2.ts:2]: 7

Comment (Dylan0916) [reply src/utils2.ts:2]: 8

Comment (Dylan0916) [reply src/utils2.ts:2]: 9

Comment (Dylan0916) [reply src/utils2.ts:2]: 10

Comment (Dylan0916) [reply src/utils2.ts:2]: 11

Comment (dehao0916) [reply src/utils2.ts:2]: 12

Comment (dehao0916) [reply src/utils2.ts:2]: 13

Comment (Dylan0916) [reply src/utils2.ts:2]: 12

**Comment by Dylan0916** [comment]
test1

File: src/utils2.ts (hunk 1 above)
Comment (Dylan0916) [reply src/utils2.ts:2]: 13

**Pull request closed without merging**